Toolset/libraries/revbackscriptlibrary.livecodescript

```diff
@@ -2309,42 +2194,105 @@ private function revSEStackIsScriptEditor pStackname
    return false
 end revSEStackIsScriptEditor
 
+on revSEHidePalettes
+   cancel sPalettesMsgID
+   send "revSEDelayTogglePalettes false" to me in 200 milliseconds
```

Comment:
@peter-b No particular reason other than it allows time for other scripts to toggle the palettes shown and hidden multiple times without doing anything and still presents the palettes or hides them depending on the final state within an imperceptible time for the user.